CMakeLists.txt

```diff
@@ -517,6 +517,13 @@ if(GUI)
     )
 endif()
 
+if(WUI)
+  target_sources(
+    firmware PRIVATE src/wui/wui.c src/wui/wui_api.c src/wui/http/fs.c src/wui/http/httpd.c
+    )
+  target_compile_definitions(firmware PRIVATE BUDDY_ENABLE_WUI)
+endif()
+
```

Comment:
It would be great to have the WUI as a separate library. Have you considered it? 

The plan is to do the same with GUI etc., but those have to be refactored first.

Reply:
in the next version of WUI, it is already configurable library. one can build with or without WUI. Same with GUI, just wait for pull request ;)

Reply:
for this merge no plan to change the sub-module for LWIP, may be considered in future

Reply:
i will check the ilb/marlin

Reply:
@joshymjose good enough for now